apps/frontend/src/libs/modules/api/base-http-api.ts

```diff
@@ -1,3 +1,137 @@
+// import { ServerErrorType } from "~/libs/enums/enums.js";
```

Comment:
I believe this comment should be removed 😅

Reply:
Ohhh, yeah, sure, sorry